### DISCUSSION TITLE
Why does RingBufferLogEvent not clear threadName field in clear() method?

### DISCUSSION THREAD

**Discussion by piecebook** [apache/logging-log4j2#2233] [Q&A] [answered]
Why does RingBufferLogEvent not clear threadName field in clear() method? 
[RingBufferLogEvent](https://github.com/apache/logging-log4j2/blob/2.x/log4j-core/src/main/java/org/apache/logging/log4j/core/async/RingBufferLogEvent.java)
![image](https://github.com/apache/logging-log4j2/assets/20704443/5c5f238a-4a69-4041-ad6b-5a1f9c90f514)

**Comment by vy** [comment, marked as answer]
@piecebook, this indeed looks like a bug. Thanks for the report. I will fix it and add tests too. Created #2234.

**Reply by piecebook** [reply]
Thank you for the prompt reply.
